Sequence.java

```diff
@@ -0,0 +1,21 @@
+import java.util.Scanner;
+
+public class Sequence {
+  public static void main(String[] args) { 
+    Scanner input = new Scanner(System.in);
+    System.out.println("Enter an integer sequence(without blancs) : ");
```

Comment:
i.e. user is going to input 33 and 222 that is increasing sequence but your code recognizes it as 3 3 2 2 2 that is non-increasing sequence.